app/process/roads_bikelanes/roads_bikelanes.lua

```diff
@@ -88,4 +122,13 @@ function osm2pgsql.process_way(object)
     meta = Metadata(object),
     geom = object:as_linestring()
   })
+  for _, cycleway in pairs(cycleways) do
+    if not cycleway.onlyPresent then
```

Comment:
Was bedeutet das? > Comment

Reply:
Oder kann man den OnlyPresence helper in Bikelanes raus nehmen und hier als Filter anwenden?

Reply:
Nein, wir brauchen den auch in Bikelanes.

Reply:
Evtl. können wir es durch das Refactoring der Vererbung besser Regeln. Aber aktuell ist es nicht leicht es aus der Bikelanes.lua zu trennen.

Reply:
Lässt sich vermutlich durch den Refactor der Catgeries besser lösen